packages/javascript-mutator/src/mutators/ArrayNewExpressionMutator.ts

```diff
@@ -7,15 +7,13 @@ import { NodeMutator } from './NodeMutator';
 export default class ArrayNewExpressionMutator implements NodeMutator {
   public name = 'ArrayNewExpression';
 
-  public mutate(node: types.Node, copy: <T extends types.Node>(obj: T, deep?: boolean) => T): void | types.Node[] {
-    const nodes: types.Node[] = [];
-
+  public mutate(node: types.Node, copy: <T extends types.Node>(obj: T, deep?: boolean) => T): types.Node[] {
     if ((types.isCallExpression(node) || types.isNewExpression(node)) && types.isIdentifier(node.callee) && node.callee.name === 'Array' && node.arguments.length > 0) {
       const mutatedNode = copy(node);
       mutatedNode.arguments = [];
-      nodes.push(mutatedNode);
+      return [mutatedNode];
```

Comment:
Why is this better?

Reply:
previously we had 3 steps in order to get value:
create an array,
push element,
return array.

Right now we are doing these 3 steps in 1 step.

Reply:
about performance: after small check:
```js
function func() {
 	const k = [];   
    k.push({"hello": "hi"});
    return k;
}
```js
per = performance.now(); for(let i = 0; i < 1000000; i++) {
    func();
} console.log(performance.now() - per);
```
and
```js
function func2() {
    return [{"hello": "hi"}];
}
```
```js
per = performance.now(); for(let i = 0; i < 10000000; i++) {
    func2();    
} console.log(performance.now() - per);
```

it seems, my solution is 20 times faster